frontend/packages/kubevirt-plugin/integration-tests/v2v/rhv/v2v.rhv.import.scenario.no.start.vm.ts

```diff
@@ -1,5 +1,5 @@
 import { withResource } from '@console/shared/src/test-utils/utils';
```

Comment:
let's use the a convention of vm.wizard.scenario and have similar test cases in the same file

Reply:
Done, tests merged to the same file